src/Server.ts

```diff
@@ -1,46 +1,59 @@
 import * as bodyParser from 'body-parser';
 import * as cookieParser from 'cookie-parser';
 import * as express from 'express';
-import * as session from 'express-session';
 import * as nunjucks from 'nunjucks';
+import 'reflect-metadata';
 import * as path from 'path';
-import { router } from './routes/Routes';
+import { Container } from 'inversify';
+import { InversifyExpressServer } from 'inversify-express-utils';
+import './controllers/index';
```

Comment:
How do we plan to use this?

Reply:
This import just imports all the controllers inside the index so that we don't have to mention them all in this file. These also need to be imported once in this file so the Dependency Resolver can resolve the dependencies.

Reply:
do we have to have an index for this? can we not just point to the '/controllers' directory